models/gc_spider_baseline/config/default.yml

```diff
@@ -0,0 +1,36 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: SPIDER baseline segment vertebrae in Sagittal Spine MRI (dicom to dicom segmentation)
+  
+execute:
+- DicomImporter
+- MhaConverter
+- SpiderBaselineRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta:
+      mod: mr
```

Comment:
Doesn't the algorithm support CT and MR?
You can set `mod: '%Modality'` to extract the modality from the dicom header.

Reply:
Added